Include/internal/pycore_interp.h

```diff
@@ -242,7 +242,10 @@ struct _is {
     PyObject *audit_hooks;
     PyType_WatchCallback type_watchers[TYPE_MAX_WATCHERS];
     PyCode_WatchCallback code_watchers[CODE_MAX_WATCHERS];
-    PyContext_WatchCallback context_watchers[CONTEXT_MAX_WATCHERS];
+    struct {
+        PyContext_WatchCallback *callback;
+        PyObject *arg;
```

Comment:
Does this need a `Py_VISIT` somewhere?  I didn't see any for the other fields in this structure.